### PR TITLE
991 2 qc links

### PR DESCRIPTION
**Pull request by kkoskelin**
Screen Shot 2019-10-16 at 9 48 32 AM

### CODE REVIEW

File: web-client/src/presenter/computeds/documentDetailHelper.js [hunk 1]
```diff
@@ -44,6 +44,13 @@ export const documentDetailHelper = (get, applicationContext) => {
         return formatted;
       });
 
+    const qcWorkItemsCompleted = (formattedDocument.workItems || [])
+      .filter(wi => !wi.isInternal)
```
Comment (sutt0n) [inline web-client/src/presenter/computeds/documentDetailHelper.js]: can we use `workItem` here instead of `wi`?

**Review by sutt0n — APPROVED** [review]
Small nitpick, but looks good 👍

File: web-client/src/views/DocumentDetail/DocumentDetail.jsx [hunk 2]
```diff
@@ -215,7 +215,18 @@ export const DocumentDetail = connect(
...
+            {documentDetailHelper.formattedDocument.canEdit && (
+              <Button
+                link
+                className="margin-right-0 padding-bottom-0"
+                href={`/case-detail/${caseDetail.docketNumber}/documents/${documentDetailHelper.formattedDocument.documentId}/edit`}
+              >
+                <FontAwesomeIcon icon={['fas', 'edit']} />
+                Edit
+              </Button>
+            )}
```
Comment (wbyoko) [inline web-client/src/views/DocumentDetail/DocumentDetail.jsx]: Work Items can be edited at any time, even after QC.

I think this link should show up after QC is complete.

**Review by wbyoko — CHANGES_REQUESTED** [review]
Pretty sure the edit link doesn't show up after initial QC is complete though it should.

**Review by wbyoko — APPROVED** [review]

**Pull request merged**